specification/cognitiveservices/data-plane/FormRecognizer/preview/v1.0/FormRecognizerReceipt.json

```diff
@@ -87,21 +127,51 @@
h representing a receipt detected in the input document. </td></tr><tr><td>pages</td><th>[Integer]</th><td>List of pages where the receipt is found. Currently, only single-page receipts are supported.</td></tr><tr><td>fields</td><th>Object</th><td>Dictionary of analyzed field values. If a field is not detected, the corresponding field value will be set to null.</td></tr><tr><td>valueType</td><th>String</th><td>Data type of the field value. Currently, possible types include \\\"stringValue\\\" and \\\"numberValue\\\".</td></tr><tr><td>value</td><th>String/Number</th><td>Semantic value of the recognized field. If the text value cannot be converted, value will be set to \"\" for stringValue or null for numberValue.</td></tr><tr><td>text</td><th>String</th><td>Extracted field text in understandingResults, or a recognized word/line in recognitionResults.</td></tr><tr><td>elements</td><th>[Object]</th><td>List of references to OCR words comprising the detected field value.</td></tr><tr><td>$ref</td><th>String</th><td>JSON pointer to an OCR word element. For example, \"#/recognitionResults/0/lines/1/words/2\" refers to 3rd word in the 2nd line of the 1st page in the document.</td></tr><tr><td>lines</td><th>[Object]</th><td>An array of objects, each representing a line of recognized text. The maximum number of lines returned is 300 per page.<br />The lines are sorted top to bottom, left to right, although in certain cases proximity is treated with higher priority. As the sorting order depends on the detected text, it may change across images and OCR version updates. Thus, business logic should be built upon the actual line location instead of order.</td></tr><tr><td>words</td><th>[Object]</th><td>An array of objects, each representing a recognized word.</td></tr><tr><td>boundingBox</td><th>[Number]</th><td>The bounding box of a recognized line or word, depending on the parent object. It is represented by an array of eight numbers corresponding to four points, clockwise from the top-left corner. For image, the (x, y) coordinates are measured in pixels from the top-left corner of the image. For PDF, the (x, y) coordinates are measured in inches from the top-left corner of the each page.</td></tr><tr><td>confidence</td><th>String</th><td>This flag appears when the recognition of a word is of low confidence.</td></tr><tr><td>width</td><th>Number</th><td>The width of the image/PDF in pixels/inches, respectively.</td></tr><tr><td>height</td><th>Number</th><td>The height of the image/PDF in pixels/inches, respectively.</td></tr><tr><td>clockwiseOrientation</td><th>Number</th><td>The orientation of the image in clockwise direction, measured in degrees between [0, 360).</td></tr><tr><td>page</td><th>Integer</th><td>The 1-based page number of the recognition result.</td></tr><tr><td>unit</td><th>String</th><td>The unit used by the width, height and boundingBox properties. For images, the unit is \"pixel\". For PDF, the unit is \"inch\".</td></tr></tbody></table>",
```

Comment:
swagger will be used to generate docs at docs.ms and SDKs.  xml inside json will be terrible for downstream docs and SDKs, might cause some issue there. Could you pls put sample into sample json file instead of swagger json file?

Reply:
Got it.  This makes it nearly impossible to share Swagger with APIM documentation.  We will place the sample in the external x-ms-examples file.